interface/patient_file/printed_fee_sheet.php

```diff
@@ -550,7 +548,11 @@ function printlog_before_print() {
 if ($form_fill != 2) {   //use native browser 'print' for multipage
 $html .= "<div id='hideonprint'>
 <p>
-
+ <input type='button' class='cp-output' value='";
+        $html .= xla('Print');
+        $html .="' id='printbutton' />
```

Comment:
Nice one @ishita20 the print was looking :( where it was and appearing in the printable area. Thanks and welcome onboard.